src/test/java/seedu/address/logic/commands/DeleteShortcutCommandTest.java

```diff
@@ -0,0 +1,117 @@
+package seedu.address.logic.commands;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.UndoRedoStack;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.shortcuts.ShortcutDoubles;
+import seedu.address.model.shortcuts.UniqueShortcutDoublesList;
+
+public class DeleteShortcutCommandTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    private Model model;
+    private Model expectedModel;
+    private final String validtestingCommandWord = "list";
+    private final String validtestingShortcutWord = "l";
+    private final String invalidtestingCommandWord = "king";
+
```

Comment:
Variable names need to be in camelCase, e.g. validTestingShortcutWord. also perhaps shorten it, e.g validCommandWord, validShortcut, invalidCommandWord?

Reply:
Will correct it in the next PR as the naming has to be changed in multiple places.